typhos/positioner.py

```diff
@@ -195,18 +196,32 @@ def _clear_status_thread(self):
         self._status_thread.disconnect()
         self._status_thread = None
 
-    def _start_status_thread(self, status, timeout):
+    def _start_status_thread(
+        self,
+        status: ophyd.StatusBase,
+        timeout: float,
+        timeout_desc: str,
+    ) -> None:
         """Start the status monitoring thread for the given status object."""
         self._status_thread = thread = TyphosStatusThread(
-            status, start_delay=self._min_visible_operation,
+            status,
+            error_context="Move",
+            timeout_calc=timeout_desc,
+            start_delay=self._min_visible_operation,
             timeout=timeout,
             parent=self,
         )
-        thread.status_started.connect(self.move_changed)
+        thread.status_started.connect(self._move_started)
         thread.status_finished.connect(self._status_finished)
+        thread.error_message.connect(self._set_status_text)
         thread.start()
 
-    def _get_timeout(self, set_position: float, settle_time: float, rescale: float = 1) -> float | None:
+    def _get_timeout(
+        self,
+        set_position: float,
+        settle_time: float,
+        rescale: float = 1,
+    ) -> tuple[int | None, str]:
```

Comment:
My perpetual fear of `from __future__ import annotations` makes me fear this, but it seems to have worked.  I should start using it more